lib/improver/orographic_enhancement.py

```diff
@@ -483,12 +481,8 @@ def _create_output_cubes(self, orogenh_data, reference_cube):
                 the UK standard grid
 
         Returns:
-            (tuple): tuple containing:
-                **orogenh** (iris.cube.Cube):
-                    Orographic enhancement cube on 1 km UKPP grid (m s-1)
-                **orogenh_standard_grid** (iris.cube.Cube):
-                    Orographic enhancement cube on the UK standard grid, padded
-                    with masked np.nans where outside the UKPP domain (m s-1)
+            iris.cube.Cube:
+                Orographic enhancement cube on 1 km UKPP grid (m s-1)
```

Comment:
Again could we remove the reference to the 1 km UKPP grid please.

Reply:
Done.